webapp/src/lib/bundler.ts

```diff
@@ -29,10 +30,10 @@ export type UserOpRequest = {
 	maxPriorityFeePerGas: string;
 
 	// paymasterAndData: BytesLike;
-	// paymaster: string | undefined;
-	// paymasterVerificationGasLimit: string | undefined;
-	// paymasterPostOpGasLimit: string | undefined;
-	// paymasterData: string | undefined;
+	paymaster?: string | undefined;
```

Comment:
I think the `?` is superfluous with `string | undefined`: https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties

As far as I understand, this:

```ts
type T = {
  foo?: bar
}
```

is equivalent to:

```ts
type T = {
  foo: bar | undefined
}
```